src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Все супер!)

Декілька малих уточнень лише на рахунок оформлення:
```suggestion
    <meta
      name="viewport"
      content="width=device-width,
      initial-scale=1.0"
    >
```